### PR TITLE
asciinema: Add support for asciinema casts (they're awesome)

### PR DESCRIPTION
**Pull request by kallisti5**

### CODE REVIEW

File: themes/shijin4/layouts/partials/footer.html [hunk 1]
```diff
@@ -2,5 +2,6 @@
 	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet">
 
 	<script src="/js/navbar.js"></script>
+	<script src="/js/asciinema-player.js"></script>
```
Comment (waddlesplash) [inline themes/shijin4/layouts/partials/footer.html]: This file is probably pretty large; please only include it when it's actually needed. Same for the CSS.

**Comment by jt15s** [comment]
Is this still required? Looking at the website here: https://asciinema.org/ it seems this is a tool for recording terminal sessions, which _could_ come in handy for documentation, but would be good to see everyone else's opinion on this.

**Comment by pulkomandy** [comment]
I don't see why we need it. Looks like kallisti5 thought it looked cool, but we are not really a terminal-oriented system so we don't really have a need for this.

For the occasional list of commands in documentation, a static view seems to be better. This tool is mostly useful for demonstrating ncurses apps or other similar complex things running in Terminal, but we don't have a lot of that here.

**Comment by jt15s** [comment]
Ah ok. In that case, should someone close this PR?

**Pull request closed without merging**